cranelift-codegen/src/isa/mod.rs

```diff
@@ -90,29 +90,25 @@ mod stack;
 /// Returns a builder that can create a corresponding `TargetIsa`
 /// or `Err(LookupError::Unsupported)` if not enabled.
 macro_rules! isa_builder {
-    ($name:ident, $feature:tt) => {{
-        #[cfg(feature = $feature)]
-        fn $name(triple: Triple) -> Result<Builder, LookupError> {
-            Ok($name::isa_builder(triple))
-        };
-        #[cfg(not(feature = $feature))]
-        fn $name(_triple: Triple) -> Result<Builder, LookupError> {
-            Err(LookupError::Unsupported)
+    ($name: ident, $feature: tt, $triple: ident) => {{
```

Comment:
The name is now unuses.

Reply:
It's used line 95.

Reply:
Oops.

Reply:
There's probably a macro way to reuse $name for $feature, using the `stringify!` macro, though.

Reply:
Doing so would imply that we always keep the same for both, and modules names are more restricted than feature names.